### DISCUSSION TITLE
Google Enterprise MDM Support

### DISCUSSION THREAD

**Discussion by adepaoli** [strongswan/strongswan#673] [Ideas]
Is there someone working on extending the APK to offers managed configuration (MDM)?
We could have better support on an Enterprise level.

See: https://developer.android.com/work/managed-configurations

**Comment by tobiasbrunner** [comment]
Not to my knowledge. Maybe [importable configuration profiles](https://wiki.strongswan.org/projects/strongswan/wiki/AndroidVPNClientProfiles) are helpful to you?
